MapboxStatic/Overlay.swift

```diff
@@ -395,3 +395,29 @@ open class Path: NSObject, Overlay {
         return description
     }
 }
+
+extension JSONSerialization.WritingOptions {
+    
+    static var sortedIfAvailable: JSONSerialization.WritingOptions {
+        
+        #if os(OSX)
+        if #available(OSX 10.13, *) {
+            return [.sortedKeys]
+        }
+        #elseif os(iOS)
+        if #available(iOS 11.0, *) {
+            return [.sortedKeys]
+        }
+        #elseif os(tvOS)
+        if #available(tvOS 11.0, *) {
+            return [.sortedKeys]
+        }
+        #elseif os(watchOS)
+        if #available(watchOS 4.0, *) {
+            return [.sortedKeys]
+        }
+        #endif
+        
+        return []
+    }
```

Comment:
Although this is good for testing, it isn't really needed outside of it and might lead to a slight performance cost. Would it be better to wrap this in a `#if DEBUG` statement?